tests/foreman/api/test_host.py

```diff
@@ -717,19 +852,22 @@ def test_positive_update_os(self):
         medium.update(['operatingsystem'])
         host.operatingsystem = new_os
         host = host.update(['operatingsystem'])
-        self.assertEqual(host.operatingsystem.read().name, new_os.name)
+        assert host.operatingsystem.read().name == new_os.name
 
     @tier2
-    def test_positive_update_medium(self):
-        """Update a host with a new medium
+    def test_positive_create_and_update_medium(self, module_org, module_location):
+        """Create and update a host with a medium
 
         :id: d81cb65c-48b3-4ce3-971e-51b9dd123697
 
-        :expectedresults: A host is updated with a new medium
+        :expectedresults: A host is created and updated with expected medium
 
         :CaseLevel: Integration
         """
-        host = entities.Host().create()
+        medium = entities.Media(organization=[module_org], location=[module_location],).create()
```

Comment:
check if same available in global fixtures else worth creating there ?

Reply:
DONE.

Reply:
Seems this is not moved to fixtures! I could see the same code still exists. 

Not a blocker to merge your PR though. But you have chance to add one more golden fixture. :)